### PR TITLE
adding Mic's VC tutorial

### PR DESCRIPTION
**Pull request by angiejones**

### CODE REVIEW

File: site/docs/ssi/credential-lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+
+## Example: A Credit Score Service
+
+We have a fictitious credit scoring bureau called MoeX. MoeX will issue a credential that contains a user's credit score and they can share this VC freely with anyone. 
+
+This will be a web app which takes a user's DID and name, and then returns a VC which contains their credit score. 
+
+### Design a Schema
+
+MoeX has 2 items in its credit credential: Overall Score and Number of Defaults. This credential is only good for 30 days, but otherwise isn't revoked. So, let's create a schema for this:
```
Comment (angiejones) [inline site/docs/ssi/credential-lifecycle.md:297]: @michaelneale what about the schema makes the VC only good for 30 days?

File: site/docs/ssi/credential-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
+      "credentialSubject": {
+        "type": "object",
+        "properties": {
+          "creditScore": {
+            "type": "integer"
+         },
+          "numOfDefaults": {
```
Comment (angiejones) [inline site/docs/ssi/credential-lifecycle.md:312]: @michaelneale why is this included in the schema? you don't add anything to the credential for this.

File: site/docs/ssi/credential-lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+
+
+## Use the Credential
+
+Now you have a credential issued to a person (a holder), how does that person use that credential?
+
+They present the `credentialsJwt` to any service or app that accepts it to prove they have the qualification. If that service trusts the issuer, then they can trust the claim, and it is easy and instant to check. 
+
+This is sometimes called the [triangle of trust](https://en.wikipedia.org/wiki/Verifiable_credentials#/media/File:VC_triangle_of_Trust.svg"). 
+
```
Comment (angiejones) [inline site/docs/ssi/credential-lifecycle.md:248]: @michaelneale is this where you want to embed your image with the popup descriptions?

File: site/docs/ssi/credential-lifecycle.md (hunk 1 above)
Comment (michaelneale) [reply site/docs/ssi/credential-lifecycle.md:297]: There is nothing in the schema that can specify that, its more a conceptual design (I realised that after I wrote that) so we can either remove that or just leave it in. Expiry etc is only specified when you create the credential, schema seems to really be only about the data shape.

File: site/docs/ssi/credential-lifecycle.md (hunk 2 above)
Comment (michaelneale) [reply site/docs/ssi/credential-lifecycle.md:312]: was to show that can have an optional field.

File: site/docs/ssi/credential-lifecycle.md (hunk 3 above)
Comment (michaelneale) [reply site/docs/ssi/credential-lifecycle.md:248]: yes - I was thinking that would ultimately be on another page that is clickable, but this could link to that in future, wherever that is.

File: site/docs/ssi/credential-lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,479 @@
+---
+sidebar_position: 2
+title: Credential Lifecyle Flow
```
Comment (michaelneale) [inline site/docs/ssi/credential-lifecycle.md:3]: should we name this "credential issuance and verification" or "credential issuance and usage tutorial" to make it clear its hands on?

**Review by michaelneale — APPROVED** [review]

File: site/docs/ssi/credential-lifecycle.md (hunk 1 above)
Comment (angiejones) [reply site/docs/ssi/credential-lifecycle.md:297]: I think we should add it to show the example. Is this the format? cc: @decentralgabe 

```bash
curl -X PUT localhost:8080/v1/credentials -d '{
  "issuer": "did:key:z6MkqGF11YyzCgVr6G3dVJULjbBokSXN4J3cjSB5MQdXto4o", 
  "verificationMethodId": "did:key:z6MkqGF11YyzCgVr6G3dVJULjbBokSXN4J3cjSB5MQdXto4o#z6MkqGF11YyzCgVr6G3dVJULjbBokSXN4J3cjSB5MQdXto4o",
  "subject": "did:ion:EiClkZMDxPKqC9c-umQfTkR8vvZ9JPhl_xLDI9Nfk38w5w",
  "schemaId": "0bee9256-dab1-40c6-b6c7-6a6e0164c0c5",
  "expirationDate": "2023-09-09T19:23:24Z",
  "data": {
    "firstName": "Satoshi",
    "lastName": "Nakamoto"
  }
}'
```

Comment (angiejones) [reply site/docs/ssi/credential-lifecycle.md:297]: nvm, we have an example in our [guide](https://developer-tbd-website-site-git-credential-is-d0cc9d-tbd54566975.vercel.app/docs/ssi/create-credentials#create-verifiable-credential) already 🎉 I'll add the field to this one to drive home the 30-day expiration narrative

File: site/docs/ssi/credential-lifecycle.md (hunk 4 above)
Comment (angiejones) [reply site/docs/ssi/credential-lifecycle.md:3]: I'm trying to differentiate it from this long list of other guides that are coming as well as this [existing one](http://localhost:3000/docs/ssi/create-credentials). Any other ideas?
image

Comment (michaelneale) [reply site/docs/ssi/credential-lifecycle.md:2]: this one actually builds an external issuance service so that stands out (it is closer to how someone may deploy it in their enterprise)

Comment (michaelneale) [reply site/docs/ssi/credential-lifecycle.md:2]: "Credential Issuance service"
